GameData/RP-0/Contracts/Moon Exploration/Lunar Landing Far Side.cfg

```diff
@@ -0,0 +1,156 @@
+CONTRACT_TYPE
+{
+	name = landingMoonFarSide
+	title = Far Side Lunar Landing(Uncrewed)
+	group = MoonExploration
+
+
+	description = Design and successfully launch a craft that can achieve a soft landing on the far side of the Moon and return science data back to Earth. You may complete this mission up to 2 times.<b><color=white>NOTE: You can not select this contract and the First Crewed Lunar Landing contract. If you select the other contract while this is active, this will automatically fail.</color></b>&br;&br;<b>Number of Contracts Completed: $landingMoonFarSide_Count</b>
+	genericDescription = Achieve a soft landing on the Moon and transmit Science.
+
+	synopsis = Achieve a soft landing on the far side of the Moon and transmit Science
+
+	completedMessage = Mission Success! You have safely landed on the far side of the Moon and returned valuable science data from the surface. We now know that it is possible for a crewed craft to land there safely as well.
+
+	sortKey = 703
+
+	cancellable = true
+	declinable = true
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 2
+	maxSimultaneous = 1
+	deadline = 730 * RP1DeadlineMult()  // 2 years
+
+	targetBody = Moon
+
+
+	// ************ REWARDS ************
+	prestige = Significant   // 1.25x
+	advanceFunds = @/advance * @RP0:globalHardContractMultiplier
+	rewardScience = 0
+	rewardReputation = 50
+	rewardFunds = @/reward * @RP0:globalHardContractMultiplier
+	failureReputation = 25
+	failureFunds = @advanceFunds * 0.5
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = CompleteContract
+		type = CompleteContract
+		contractType = first_MoonFlybyUncrewed
+	}
+	REQUIREMENT
+	{
+		name = CompleteContract
+		type = CompleteContract
+		contractType = landingMoon
+		minCount = 2
+		title = Complete 'Lunar Landing' contract at least @minCount times
+	}
+	REQUIREMENT
+	{
+		name = CompleteContract
+		type = CompleteContract
+		contractType = first_MoonLandingCrewed
```

Comment:
The idea with this is to fail the contract if the crewed landing is accepted (I think), however is this the correct implementation? (copied from the normal lander contract)

Reply:
But in this case the type should be AcceptContract (or something like that) instead

Reply:
Removed the REQUIREMENT with the latest commit. If it is needed to prevent this contract from being accepted while any of the crewed contracts are accepted, I can add that.